lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/security/SoapToken.java

```diff
@@ -36,6 +38,14 @@ public SoapToken(
     public String getToken() {
         long startTime = System.nanoTime();
 
+        Span span =
+                OpenTelemetryUtil.createSpan(
+                        this.getClass(),
+                        "getToken",
+                        "Experian getToken",
+                        "_UNKNOWN",
```

Comment:
Should be `LoginWithCertificate` to be consistent with the webservice endpoints

Reply:
_UNKNOWN is for when we do not know what the http method is. It's either GET, POST, PATCH etc.. or custom in some cases SAA/RTQ. LoginWithCertificate wouldn't be a valid HTTP method and also I can't tell from the decompiled classes what the request method is.

Reply:
https://opentelemetry.io/docs/specs/semconv/http/http-spans/

Reply:
I think all SOAP calls are post under the hood actually, and they are generally un-restlike and always CUSTOM, so SAA, RTQ and in the case it is actually known `LoginWithCertificate` is the Operation or method just like SAA and RTQ